src/components/AppConfig/AuthSettings/TokenAuth.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
```

Comment:
Just to keep this PR smaller, could we get rid of token auth entirely since it's not needed atm?